docs/development-setup.md

```diff
@@ -17,8 +17,8 @@ We'll use the `oc cluster up` method, since it's very easy to use:
     ```
     $ oc cluster up
     ```
-    (If you are running into some connection issues, please flush your iptables
-    rules bufore starting the cluster with `iptables -F`)
```

Comment:
The linked documentation does not mention `iptables -F`. This works for me whenever a build fails to lookup github.com, for example. I don't think we should remove that. Maybe:
> flush your iptables rules before starting the cluster with `iptables -F`, for further assistance see...